<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QDialog" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1680000</width>
       <height>1678000</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>程序写的都队</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="easyButton">
           <property name="minimumSize">
            <size>
             <width>232</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>AIGDT</family>
             <pointsize>14</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>EASY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="medium1Button">
           <property name="font">
            <font>
             <family>AIGDT</family>
             <pointsize>14</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>MEDIUM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hardButton">
           <property name="font">
            <font>
             <family>AIGDT</family>
             <pointsize>14</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>HARD</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="6">
       <widget class="QLabel" name="label_image">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>374</width>
          <height>556</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_logo">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>50</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Copperplate Gothic Bold</family>
          <pointsize>20</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>              SUDOKU</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>widget</zorder>
     <zorder>label_image</zorder>
     <zorder>label_2</zorder>
     <zorder>label</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>label_logo</zorder>
     <zorder>verticalSpacer_2</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
